addons/point_of_sale/static/tests/tours/utils/ticket_screen_util.js

```diff
@@ -109,7 +109,7 @@ export function clickBackToMainTicketScreen() {
         },
     ];
 }
-export function confirmRefund() {
+export function clickConfirmRefund() {
```

Comment:
there's no need the make the name longer. `confirmRefund` conveys the meaning very well already. The fact that an element in the ui will be clicked is obvious, since, `confirmRefund` is clearly an action